Caltech-IPAC/firefly
# IRSA-1596: Saved table contains hidden columns, i.e.  ROW_IDX and ROW_NUM

https://caltech-ipac.atlassian.net/browse/IRSA-1596
Also:
- remove DataSetParser as part of ongoing code cleanup.
- fix IRSA-1601

To test:
- build firefly 
- catalog search
- save table by clicking on the 'diskette' icon

You should not see hidden columns like ROW_IDX and ROW_NUM anymore.
